server/src/main/resources/routes

```diff
@@ -65,7 +65,6 @@ GET           /fixed-points
 POST          /fixed-points                                                           controllers.application.StaffingController.saveFixedPoints
 POST          /shifts/save                                                            controllers.application.ShiftsController.saveShifts
 POST          /shifts/update/:previousName                                            controllers.application.ShiftsController.updateShift(previousName: String)
```

Comment:
To be RESTful the end points here should be:
`POST /shifts                 controllers.application.ShiftsController.saveShifts`
`PUT /shifts/:previousName    controllers.application.ShiftsController.updateShift(previousName: String)`